src/community/release/ext-wps-longitudinal-profile.xml

```diff
@@ -0,0 +1,16 @@
+<assembly>
+  <id>wps-longitudinal-profile</id>
```

Comment:
make sure to append -plugin to the id:
<id>wps-longitudinal-profile-plugin</id>

If my memory serves me right, in the past I have defined a new module using an id without the -plugin suffix and it was not showing up in the download list.

Reply:
done